coalib/settings/ConfigurationGathering.py

```diff
@@ -385,6 +385,20 @@ def gather_configuration(acquire_settings,
     local_bears, global_bears = fill_settings(sections,
                                               acquire_settings,
                                               log_printer)
+
+    exclude = False
+    for x in arg_list:
```

Comment:
Hm the argparser docs https://docs.python.org/3/library/argparse.html#nargs seems able to correctly parse something like that as a list. Maybe you must use `nargs='*'` for it to works.